### ISSUE TITLE
The previously disabled tasks were restored to a runnable state after migrating the job from version 3.x to 4.0.0-beta2

### ISSUE THREAD

**Issue by cobolbaby** [distribworks/dkron#1466]
**Describe the bug**

**To Reproduce**

![2024-02-07 09-55-40屏幕截图](https://github.com/distribworks/dkron/assets/11460848/c3837570-9325-477d-9718-32255fb0afd7)

```bash
curl -v 'https://infra-datajob.ipt.inventec.net/v1/jobs' > backup.json
curl -v http://localhost:6230/v1/restore --form 'file=@backup.json'
```

![2024-02-07 09-56-04屏幕截图](https://github.com/distribworks/dkron/assets/11460848/2052594b-ae75-4712-99e0-78546a12fe9f)

**Expected behavior**

**Screenshots**

** Specifications:**
 - OS: [e.g. linux]
 - Version [e.g. 2.0.1]

**Additional context**

**Comment by cobolbaby** [comment]
It seems to be just a UI issue. The API response is normal.

![2024-02-07 11-07-49屏幕截图](https://github.com/distribworks/dkron/assets/11460848/28a96918-30aa-4c76-84ef-fef562f909d0)

**Comment by vcastellm** [comment]
Nice catch, thanks for reporting, I'll take a look

**Comment by vcastellm** [comment]
Fixed in #1467
